packages/suite/src/views/settings/general/Tor.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { Switch } from '@trezor/components';
+import { useAnalytics, useSelector } from '@suite-hooks';
+import { ActionColumn, SectionItem, TextColumn } from '@suite-components/Settings';
+import { Translation } from '@suite-components';
+import { useAnchor } from '@suite-hooks/useAnchor';
+import { SettingsAnchor } from '@suite-constants/anchors';
+
+export const Tor = () => {
+    const analytics = useAnalytics();
+    const { tor } = useSelector(state => ({
+        tor: state.suite.tor,
```

Comment:
That `tor` seams confusing to me, maybe something like `isTorEnable` would be more easy to read and understand.